pkgs/development/python-modules/klaus/default.nix

```diff
@@ -52,6 +53,15 @@ buildPythonPackage rec {
   # Needs to set up some git repos
   doCheck = false;
 
+  # TODO: remove when this is upstreamed
+  patches = [
+    (fetchpatch {
+      name = "distutils.patch";
+      url = "https://github.com/jonashaag/klaus/commit/d50d2aab97fd86c11f3b5a4c1ecbcf1e085f395f.patch";
```

Comment:
can you can a link to PR in a comment above ?

Reply:
PR has been merged, now it's just next release. Comment updated.